### PR TITLE
Added ZoneMapKey & Auth settings for Internet Settings

### PR DESCRIPTION
**Pull request by flacostec**
I've added ZoneMapKey & ZoneAuthSettings registry key enumeration to know URLs within the different zones. 

Also, I've included an enumeration of the "User Authentication: Logon (1A00)" settings that if misconfigured, can allow an adversary to steal NetNTLMv2 easily. 

[More information @ Microsoft website](https://support.microsoft.com/en-ca/help/182569/internet-explorer-security-zones-registry-entries-for-advanced-users)

Thanks,

### CODE REVIEW

**Comment by HarmJ0y** [comment]
Thanks for the contribution!

File: Seatbelt/Commands/Windows/InternetSettingsCommand.cs [hunk 1]
```diff
@@ -54,6 +54,89 @@ public InternetSettingsCommand(Runtime runtime) : base(runtime)
...
+                                                {0x00020000, "Automatic logon only in the Intranet zone"},
+                                                {0x00030000, "Anonymous logon"}
+                                            };
+
+            for (int i = 0; i <= 4; i++)
+            {
+                var zoneSettings = RegistryUtil.GetDwordValue(RegistryHive.LocalMachine, @"Software\Policies\Microsoft\Windows\CurrentVersion\Internet Settings\Zones\" + i.ToString(), "1A00");
+                if (zoneSettings != null)
+                {
+                    WriteHost(zoneMapKeys.AsEnumerable().Single(l => l.Key == i.ToString()).Value + "\tSettings: " + zoneAuthSettings.AsEnumerable().Single(l => l.Key == zoneSettings).Value);
```
Comment (leechristensen) [inline Seatbelt/Commands/Windows/InternetSettingsCommand.cs:137]: @flacostec any chance you could remove the call to WriteHost and replace it with a TextFormatter similar to this: 
https://github.com/GhostPack/Seatbelt/blob/master/Seatbelt/Commands/Windows/OSInfoCommand.cs#L140-L180

Comment (leechristensen) [reply Seatbelt/Commands/Windows/InternetSettingsCommand.cs:137]: Actually, nevermind, working on it right now

Comment (flacostec) [reply Seatbelt/Commands/Windows/InternetSettingsCommand.cs:137]: Okay good, let me know if you want me to do it.

**Comment by leechristensen** [comment]
@flacostec would you mind testing #26 to make sure I didn't screw anything up?

**Comment by flacostec** [comment]
@leechristensen just tested `InternetSettings` and `-group=all` and it worked without any errors. Had to downgrade the project to .NET 3.5 although.

**Comment by leechristensen** [comment]
Merged, thanks again for the contribution/testing! Regarding .NET 3.5, it should work in both 3.5/4.0 without issue (just need to change what you compile to). I switched it back to 3.5 to ensure it was still compatible with 3.5

**Pull request merged**